packages/core/src/utilities/sortImageIdsAndGetSpacing.ts

```diff
@@ -157,14 +157,14 @@ export default function sortImageIdsAndGetSpacing(
   // If possible, we use the sliceThickness, but we warn about this dicom file
   // weirdness. If sliceThickness is not available, we set to 1 just to render
   if (zSpacing === 0 && !strictZSpacingForVolumeViewport) {
-    if (sliceThickness && spacingBetweenSlices) {
+    if (!sliceThickness && spacingBetweenSlices) {
```

Comment:
Why do you care about slice thickness at all - just order it starting at the preferred item, then moving to the least preferred.  So maybe
if( spacingBetweenSclies ) {
...
else if ( sliceThickness)


That is, ignore the sliceThickness in the first test, OR put the slice thickness version first.

Reply:
Done